### PR TITLE
RzShell: convert ou command

### PR DESCRIPTION
**Pull request by ret2libc**

### CODE REVIEW

File: librz/core/cmd/cmd_open.c [hunk 1]
```diff
@@ -1588,3 +1567,25 @@ RZ_IPI RzCmdStatus rz_open_close_all_handler(RzCore *core, int argc, const char
...
+			if (bf) {
+				rz_core_bin_raise(core, bf->id);
+				rz_core_block_read(core);
+				RZ_LOG_INFO("Switched to fd %d (%s)\n", fdnum, bf->file);
+			}
+			return RZ_CMD_STATUS_OK;
+		}
+	}
+	RZ_LOG_ERROR("Could not find any opened file with fd %d\n", fdnum);
+	return RZ_CMD_STATUS_ERROR;
```
Comment (wargio) [inline librz/core/cmd/cmd_open.c]: this is just a comment, but this code does not really work. we need to fix this in another commit.

**Review by wargio — APPROVED** [review]

File: librz/core/cmd/cmd_open.c (hunk 1 above)
Comment (ret2libc) [reply librz/core/cmd/cmd_open.c]: what do you mean? What does not work?

Comment (wargio) [reply librz/core/cmd/cmd_open.c]: try it. just does not really work. print a function, change fd and then try again with another function

Comment (ret2libc) [reply librz/core/cmd/cmd_open.c]: ah if you mean analysis I didn't test it. It should work though at a more low level.

**Review by XVilka — APPROVED** [review]

**Pull request merged**